client/my-sites/checkout/upsell-nudge/premium-plan-upgrade-upsell/index.jsx

```diff
@@ -0,0 +1,244 @@
+/**
+ * External dependencies
+ */
+import React, { PureComponent } from 'react';
+import Gridicon from 'calypso/components/gridicon';
+import formatCurrency from '@automattic/format-currency';
+
+/**
+ * Internal dependencies
+ */
+import { CompactCard, Button } from '@automattic/components';
+import DocumentHead from 'calypso/components/data/document-head';
+import PageViewTracker from 'calypso/lib/analytics/page-view-tracker';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+/**
+ * Image dependencies
+ */
+import premiumThemesImage from 'calypso/assets/images/illustrations/themes.svg';
+
+export class PremiumPlanUpgradeUpsell extends PureComponent {
+	render() {
+		const { receiptId, translate } = this.props;
+
+		const title = translate( 'Checkout ‹ Plan Upgrade', {
+			comment: '"Checkout" is the part of the site where a user is preparing to make a purchase.',
+		} );
+
+		return (
+			<>
+				<PageViewTracker
+					path="/checkout/:site/offer-plan-upgrade/:upgrade_item/:receipt_id"
+					title={ title }
+					properties={ { upgrade_item: 'premium' } }
+				/>
+				<DocumentHead title={ title } />
+				{ receiptId ? (
+					<CompactCard className="premium-plan-upgrade-upsell__card-header">
+						{ this.header() }
+					</CompactCard>
+				) : (
+					''
+				) }
+				<CompactCard className="premium-plan-upgrade-upsell__card-body">
+					{ this.body() }
+				</CompactCard>
+				<CompactCard className="premium-plan-upgrade-upsell__card-footer">
+					{ this.footer() }
+				</CompactCard>
+			</>
+		);
+	}
+
+	header() {
+		const { translate } = this.props;
+
+		return (
+			<header className="premium-plan-upgrade-upsell__small-header">
+				<h2 className="premium-plan-upgrade-upsell__title">
+					{ translate( 'This is a one time offer just for you' ) }
+				</h2>
+			</header>
+		);
+	}
+
+	body() {
+		const {
+			translate,
+			planRawPrice,
+			planDiscountedRawPrice,
+			currencyCode,
+			hasSevenDayRefundPeriod,
+		} = this.props;
+		return (
+			<>
+				<h2 className="premium-plan-upgrade-upsell__header">
+					{ translate( 'Get access to Premium designs with this special offer', {
+						components: { br: <br /> },
+					} ) }
+				</h2>
+
+				<div className="premium-plan-upgrade-upsell__column-pane">
+					<div className="premium-plan-upgrade-upsell__column-content">
+						<p>
+							<b>
+								{ translate(
+									'According to Google, design is possibly the best investment you can make for your website.'
+								) }
+							</b>
+						</p>
+						<p>
+							{ translate(
+								"Wouldn't you like to make a great first impression in those three seconds?"
+							) }
+						</p>
+						<p>
+							{ translate(
+								'That’s exactly why we’ve partnered with some of the world’s greatest designers to offer nearly 200 high-end designs that you can use to make your site looks incredible.'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'That's exactly why we've partnered with some of the world's greatest designers to offer high-end designs that you can use to make your site look incredible.' )` *ES Score: 8*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/1-click-premium-bump)